src/main/java/org/alliancegenome/curation_api/jobs/executors/LoadFileExecutor.java

```diff
@@ -210,7 +210,7 @@ public void runCleanup(DiseaseAnnotationService service, BulkLoadFileHistory his
 		for (Long id : idsToRemove) {
 			try {
 				String loadDescription = dataProviderName + " association bulk load (" + md5sum + ")";
-				service.deprecateOrDeleteAssociation(id, false, loadDescription, true);
+				service.deprecateOrDeleteAssociation(id, false, loadDescription, false);
```

Comment:
This should have been false as want to delete rather than deprecate when doing cleanup of association loads - only want to deprecate if the method is called by the gene/allele load cleanup code